packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -156,6 +157,8 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         this.registry.set(plugin.model.id, plugin);
         if (plugin.pluginPath && Array.isArray(plugin.rawModel.activationEvents)) {
             const activation = () => this.loadPlugin(plugin, configStorage);
+            // an internal activation event is a subject to change
+            this.setActivation(`onPlugin:${plugin.model.id}`, activation);
```

Comment:
I can see four activations:
```
onPlugin:test.workspace-contains-glob
workspaceContains:**/core/package.json
onPlugin:test.workspace-contains-path
workspaceContains:packages/core/package.json
```

Is this expected?

Reply:
yes, 2 from package.json and 2 to activate a plugin by id

Reply:
OK, thanks! Just wanted to make sure we all aware of this 😊